website/docusaurus.config.js

```diff
@@ -0,0 +1,126 @@
+// const versions = require('./versions.json');
+
+module.exports = {
+  title: 'Graphback',
+  tagline: 'GraphQL API for your client and server',
+  url: 'https://graphback.dev',
+  baseUrl: '/',
+  favicon: 'img/favicon.ico',
+  organizationName: 'aerogear', // Usually your GitHub org/user name.
+  projectName: 'graphback', // Usually your repo name.
+  themeConfig: {
+    disableDarkMode: true,
+    prism: {
+      // eslint-disable-next-line
+      theme: require('prism-react-renderer/themes/github'),
+      // theme: require('prism-react-renderer/themes/dracula'),
+      defaultLanguage: 'javascript',
+    },
+    navbar: {
+      title: 'Graphback',
+      logo: {
+        alt: 'Graphback Logo',
+        src: 'img/logo.png',
+      },
+      links: [
+        {
+          to: 'docs/gettingstarted',
+          activeBasePath: 'docs',
+          label: 'Docs',
+          position: 'left',
+          items: [
+            {
+              label: '0.12.x',
+              to: 'docs/gettingstarted',
+            },
+            // ...versions.slice(1).map((version) => ({
+            //   label: version,
+            //   to: `docs/${version}/introduction`,
+            // }))
+            // {
+            //   label: 'Master/Unreleased',
+            //   to: 'docs/next/introduction',
+            // },
+          ],
+        },
+        {
+          to: 'docs/gettingstarted',
+          label: `v0.12.x`,
+          position: 'right',
+        },
+        {
+          href: 'https://github.com/facebook/docusaurus',
```

Comment:
Change this to graphback

Reply:
Oh haven't pushed that yet. I have fixed that already